project.godot

```diff
@@ -114,6 +114,11 @@ F5={
 "events": [ Object(InputEventKey,"resource_local_to_scene":false,"resource_name":"","device":0,"alt":false,"shift":false,"control":false,"meta":false,"command":false,"pressed":false,"scancode":16777248,"physical_scancode":0,"unicode":0,"echo":false,"script":null)
  ]
 }
+progress_dialogue={
+"deadzone": 0.5,
+"events": [ Object(InputEventKey,"resource_local_to_scene":false,"resource_name":"","device":0,"alt":false,"shift":false,"control":false,"meta":false,"command":false,"pressed":false,"scancode":16777221,"physical_scancode":0,"unicode":0,"echo":false,"script":null)
+ ]
+}
```

Comment:
I honestly don't know why this is here... Maybe a few theories.
I was working on dialogues at some point, but in a separate branch. Maybe Godot just got confused locally?
That or it has to do with some older commit I did in develop. That wouldn't explain why it changed, though...